### DISCUSSION TITLE
Different page IDs - GraphQL query & actual page ID

### DISCUSSION THREAD

**Discussion by rhvitfeldt** [requarks/wiki#5951] [Error / Bug Report] [answered]
Hi,

We get different page IDs returned when doing a GraphQL search query compared to the actual page ID. For example we do a query which returns page ID 94 for a page that, in the admin GUI of the wiki, actually has ID 52. We use "Database - PostgreSQL" as Search Engine. Changing to "Database - Basic" returns the correct ID. But we need to be able to search and get hits on content. What can we do to solve it?

Another post with the exact same issue: https://stackoverflow.com/questions/73407808/wikijs-graphql-api-returning-2-different-ids-for-the-same-page

**Comment by NGPixel** [comment, marked as answer]
Search result IDs are not page IDs, they are... search result IDs. The reason the "DB - Basic" engine returns identical IDs is because there's no search index, it queries the pages table directly.

It's not possible to get the page ID at the moment. This is coming in v3.
